projects/bjerk-conf/resources/github/providers.ts

```diff
@@ -1,9 +1,10 @@
 import * as github from '@pulumi/github';
 import { getToken } from 'get-pulumi-secret';
+import { bjerkBotGitHubToken } from '../../../bjerk-io/src/stack-refs';
```

Comment:
I think we should copy this, so that there's a stack ref in each project.

Reply:
Done. There is now a stack ref in bjerk-conf as well